source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,43 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
+method. If no document is found, ``findOne`` returns the ``null``. If a
+document is found, ``findOne`` returns that document as a Object.
```

Comment:
Grammatical issue:
* If a document is found, findOne returns that document as a Object. Should read: If a document is found, findOne returns that document as an Object.